app/controllers/api/case_requests_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::CaseRequestsController < ApplicationController
+  before_action :authenticate!
+
+  def index
```

Comment:
I'm wondering if it'd make more sense to create a custom endpoint for displaying these counts and leave the `index` action to page through all case requests (similar to `Sf311CasesController`). Also, it may be helpful to add a `show` action for fetching individual cases; not 100% sure about whether it will be necessary though.

Reply:
Yep, makes sense to me. Just updated it to `/case_requests/counts` instead of the index route.